tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -2903,25 +2911,32 @@ <h2>Create a distance feature</h2>
           } else {
             item[inferenceValueStr] =
               self.examplesAndInferences[index].inferences[
-                self.examplesAndInferences[index].inferences.length - 1][
+                self.examplesAndInferences[index].inferences.length - 1][0][
                   0].score;
           }
 
           // If there is a label vocab for classification, set the string
           // inferred label for each example.
+          // TODO index to more than one model
```

Comment:
can you change the TODOs all over to TODO(tolgab) to tag them. even though me or someone else might implement them.